actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/service/UserProfileReadService.java

```diff
@@ -122,8 +124,18 @@ public Response getUserProfileData(Request actorMessage) {
     addFlagValue(result);
     // For Backward compatibility , In ES we were sending identifier field
     result.put(JsonKey.IDENTIFIER, userId);
+    Map<String, Object> result1 = new HashMap<>();
```

Comment:
No need to create new map, in existing only u can replace the usertype and subtype. 
profile usertype need not be removed. 
new string creation not required and need not type cast to string as it may also be null

Reply:
But we need variables to get profileusertype.usertype and profileusertype.usersubtype right? or is there any another way to display contents in profileusertype?